apps/mull-ui/src/app/pages/profile/user-profile/user-profile.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+  faCog,
+  faLeaf,
+  faPencilAlt,
+  faUserFriends,
+  faUserPlus,
+} from '@fortawesome/free-solid-svg-icons';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import ProfileHeader from '../../../components/profile-header/profile-header';
+import SettingsButton from '../../../components/settings-button/settings-button';
+import './user-profile.scss';
+
+export interface userProfileProps {
+  joinDate?: string;
+  friendRequestCount?: number;
+}
+
+export const UserProfilePage = ({ joinDate, friendRequestCount }: userProfileProps) => {
+  return (
+    <div>
```

Comment:
Like I mentioned already, you could add the page-container class to this div and remove the div from `app.tsx`